home/tests/test_content_import_export.py

```diff
@@ -990,8 +1029,11 @@ def test_web_content(self, impexp: ImportExport) -> None:
         )
 
         orig = impexp.get_page_json()
+        print(orig)
         impexp.export_reimport()
         imported = impexp.get_page_json()
+        print("|||")
+        print(imported)
```

Comment:
I don't think we need these print statements anymore

Reply:
Weird, I don't see them on my end, will try figure out whats up